README.md

```diff
@@ -123,19 +123,6 @@ You can alter the variables in `chart/gatekeeper-operator/values.yaml` to custom
 
 ### Uninstallation
 
-Before uninstalling Gatekeeper, be sure to clean up old `Constraints`, `ConstraintTemplates`, and
-the `Config` resource in the `gatekeeper-system` namespace. This will make sure all finalizers
-are removed by Gatekeeper. Otherwise the finalizers will need to be removed manually.
-
-#### Before Uninstall, Clean Up Old Constraints
```

Comment:
I wonder if we should leave all the finalizer related stuff in the readme for few releases? Until users are off the old releases. Maybe add a note for `[Only applicable to v3.1.0-beta.7 and earlier]`

Reply:
We can just sit on this PR until we are comfortable if that's the concern.

Some day... versioned documentation.

Reply:
SGTM